test/geojson.test.js

```diff
@@ -54,53 +54,55 @@ describe('geojson', function () {
         });
     });
 
-    describe('#multipoint', function () {
-        it('should return coordinates, properties, and type', function () {
-            const result = geojson.multipoint(gjMultiPoint);
-
-            const idealResult = {
-                geometries: [
-                    [[2.2945064306259155, 48.85825817805569]],
-                    [[2.312525510787964, 48.854991630542706]],
-                ],
-                properties: [
-                    {
-                        name: 'MultiPoint-Name',
-                    },
-                    {
-                        name: 'MultiPoint-Name',
-                    },
-                ],
-                type: 'MULTIPOINT',
-            };
-
-            expect(result).to.eql(idealResult);
-        });
-    });
-
-    describe('#multipointZ', function () {
-        it.skip('should return coordinates, properties, and type', function () {
-            const result = geojson.multipointZ(gjMultiPointZ);
-
-            const idealResult = {
-                geometries: [
-                    [[2.2945064306259155, 48.85825817805569, 75]],
-                    [[2.312525510787964, 48.854991630542706, 5]],
-                ],
-                properties: [
-                    {
-                        name: 'MultiPointZ-Name',
-                    },
-                    {
-                        name: 'MultiPointZ-Name',
-                    },
-                ],
-                type: 'MULTIPOINTZ',
-            };
-
-            expect(result).to.eql(idealResult);
-        });
-    });
+    // Multipoint is currently not supported
+    // describe('#multipoint', function () {
+    //     it('should return coordinates, properties, and type', function () {
+    //         const result = geojson.multipoint(gjMultiPoint);
+
+    //         const idealResult = {
+    //             geometries: [
+    //                 [[2.2945064306259155, 48.85825817805569]],
+    //                 [[2.312525510787964, 48.854991630542706]],
+    //             ],
+    //             properties: [
+    //                 {
+    //                     name: 'MultiPoint-Name',
+    //                 },
+    //                 {
+    //                     name: 'MultiPoint-Name',
+    //                 },
+    //             ],
+    //             type: 'MULTIPOINT',
+    //         };
+
+    //         expect(result).to.eql(idealResult);
+    //     });
+    // });
+
+    // Multipoint is currently not supported
+    // describe('#multipointZ', function () {
+    //     it.skip('should return coordinates, properties, and type', function () {
```

Comment:
Pro-tip: you can disable tests like this line `it.skip` and it will look like the following blue line

![image](https://user-images.githubusercontent.com/8754/98041342-9e3df700-1dd6-11eb-88a1-e6e1e8f7e947.png)